### ISSUE TITLE
Redirect raises FunctionClauseError

### ISSUE THREAD

**Issue by te-chris** [smpallen99/coherence#144]
I've set the following redirect in the `redirects.ex` file:
```
 def registration_create(conn, _), do: redirect(conn, dashboard_path(conn, :index))
```
And it is raising this error:
```
FunctionClauseError at POST /registrations
no function clause matching in Phoenix.Controller.redirect/2
```

On closer inspection, it looks like it's because redirect expects a list now in phoenix (source from the exception page):
```
297 def redirect(conn, opts) when is_list(opts) do
298    url  = url(opts)
299    html = Plug.HTML.html_escape(url)
300    body = "<html><body>You are being <a href=\"#{html}\">redirected</a>.</body></html>"
```

When you set the path as the `to:` key it works, so looks like it may just need a doc update.

**Comment by smpallen99** [comment]
Agree. Are you up for submitting a PR for this? I'm behind on my open source and could really use the help!

**Comment by te-chris** [comment]
I can try :) I haven't got much experience committing to OSS, but I'll try!

**Comment by smpallen99** [comment]
its very easy. Just fork my repo. clone the repo from your github account, solve the problem and submit it back to the repo on your account. when you view on github, there will be a link to generate a pull request. It will send me a notification. I'll review and push a button to merge if I'm happy with it, or create some review comments for you to make changes. Great experience...

**Comment by pdgonzalez872** [comment]
Hi @te-chris ! Was this issue resolved? I believe so. If so, could you please close it? Thanks!
